### PR TITLE
AArch64 FC news item

### PR DESCRIPTION
**Pull request by gernotheiser**
Add a news item on the completion of the refinement proof for AArch64.

### CODE REVIEW

**Comment by gernotheiser** [comment]
Should this point to the proof repo?

**Comment by gernotheiser** [comment]
validator fails seem to result from RTX's humanity check

File: news/2024.html [hunk 1]
```diff
@@ -30,6 +30,34 @@
...
 -->
 
+<div class="news-item tag-link" id="aarch64-fc">
+  <a href="#aarch64-fc"><span class="tag-span"># </span></a> 19 Apr
+  2024: Functional correctness proved for AArch64
+</div>
+<div>
+ <p>
+
+   <a href="https://www.rtx.com/">
```
Comment (june-andronick) [inline news/2024.html:40]: This is a wrong link. I'm not sure we need a link on the image for this one so maybe just remove it.

File: news/2024.html [hunk 2]
```diff
@@ -30,6 +30,34 @@
...
+     <img src="/images/Proof-FC.svg"
+	  style="width: 20%;  padding-left:10px;  float:right"
+	  alt="Functional correctness proof"/>
+   </a>
+
+   We are extremely pleased to announce that the functional
+   correctness proof for seL4 on the 64-bit Arm architecture
+   (AArch64) is complete!<br/>
```
Comment (june-andronick) [inline news/2024.html:48]: I would suggest adding a link to https://proofcraft.systems/news-2024/#2024-04-01

**Review by june-andronick — APPROVED** [review]
Thanks @gernotheiser . I have added some comments. I'm travelling so hard to do a preview so I'll have to trust you on that. I also see a number of links issues that I guess should be sorted. My only 2 comments are to remove the link to rtx in the picture and add a link to Proofcraft's news item. Thanks

File: news/2024.html (hunk 1 above)
Comment (gernotheiser) [reply news/2024.html]: C&P error

File: news/2024.html (hunk 2 above)
Comment (gernotheiser) [reply news/2024.html:46]: done

**Pull request merged**